packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/BackgroundImageDemo/BackgroundImageDemo.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 import { BackgroundImage, BackgroundImageProps, BackgroundImageSrc } from '@patternfly/react-core';
+/* eslint-disable @typescript-eslint/no-var-requires, camelcase */
 const pfbg_1200 = require('../LoginPageDemo/images/pfbg_1200.jpg');
 const pfbg_768 = require('../LoginPageDemo/images/pfbg_768.jpg');
 const pfbg_768_2x = require('../LoginPageDemo/images/pfbg_768@2x.jpg');
 const pfbg_576 = require('../LoginPageDemo/images/pfbg_576.jpg');
 const pfbg_576_2x = require('../LoginPageDemo/images/pfbg_576@2x.jpg');
+/* eslint-enable @typescript-eslint/no-var-requires */
```

Comment:
I'm wondering why we have the no-var-requires rule? We appear to comment it out everywhere it's being used.

Reply:
Yep we're removing this now - thanks for calling this out!